examples/KeyboardandMouseControl/KeyboardandMouseControl.ino

```diff
@@ -0,0 +1,99 @@
+/*
+  This example has been moved to File > Examples > USB > KeyboardAndMouseControl
+  https://github.com/arduino/Arduino/tree/master/build/shared/examples/09.USB/KeyboardAndMouseControl
+*/
+
```

Comment:
```suggestion
```
This makes no sense. The example has not been moved to **File > Examples > USB > KeyboardAndMouseControl**. The example has always been there. The whole point of your pull request is to move (or copy) that sketch *FROM* **File > Examples > USB > KeyboardAndMouseControl** _TO_ **File > Examples > Mouse > KeyboardAndMouseControl**.

There is no benefit to providing a link to the copy of the sketch in the arduino/Arduino repository. If they saw that comment, the user is already looking at the sketch.